website/docs/configuration/outputs.html.md

```diff
@@ -96,7 +103,7 @@ Sensitive output values are still recorded in the
 to access the state data. For more information, see
 [_Sensitive Data in State_](/docs/state/sensitive-data.html).
 
-## Output Dependencies
+### `depends_on` — Manual Output Dependencies
```

Comment:
I think we've used the term "explicit" rather than "manual" when expressing this idea elsewhere.

Interestingly, when I went looking for examples to confirm I found some further interesting inconsistencies:

* In [Resource Dependencies](https://github.com/hashicorp/terraform/blob/cb4f3004daacf4382a807673178a5b0a571e01f7/website/docs/configuration/resources.html.md#resource-dependencies) we use the term "explicit" only in passing at the very end of the section in the sentence around the link for the `depends_on` meta-argument.
* [When describing `depends_on`](https://github.com/hashicorp/terraform/blob/cb4f3004daacf4382a807673178a5b0a571e01f7/website/docs/configuration/resources.html.md#depends_on-hidden-resource-dependencies) we instead use the term "hidden dependency", which makes sense if you think of it from the provider's perspective (it's a dependency in the remote system that the provider can't see), but I feel funny about that term from a user's standpoint since it may suggest that it's hidden _from the user_.

I'm not sure what's best here... I've tended to use the term "explicit" myself (with "implicit dependencies" on the other hand being the ones we infer automatically from references), but I expect I got that from an earlier generation of these docs, and it seems like the more explicit uses of that term have phased out as we've edited other sections. :thinking:

Reply:
Thank you for digging into that. Hmm... I'm inclined to re-word some of the stuff in the resources page in favor of "explicit"... I'll push another commit in a moment.